sdk/python/tests/symbol/test_TransactionFactory.py

```diff
@@ -1,94 +1,53 @@
 import unittest
 from binascii import hexlify
 
-from symbolchain.CryptoTypes import PublicKey, Signature
-from symbolchain.sc import AccountRestrictionFlags, MosaicRestrictionType, NetworkType, TransactionType, UnresolvedAddress
+from symbolchain.CryptoTypes import PublicKey
+from symbolchain.sc import AccountRestrictionFlags, LinkAction, NetworkType, TransactionType, UnresolvedAddress
 from symbolchain.symbol.IdGenerator import generate_mosaic_id, generate_namespace_id
 from symbolchain.symbol.Network import Address, Network
 from symbolchain.symbol.TransactionFactory import TransactionFactory
 
+from ..test.BasicTransactionFactoryTest import BasicTransactionFactoryTest
 from ..test.NemTestUtils import NemTestUtils
 
 FOO_NETWORK = Network('foo', 0x54)
 TEST_SIGNER_PUBLIC_KEY = NemTestUtils.randcryptotype(PublicKey)
 
 
-class TransactionFactoryTest(unittest.TestCase):
-	# pylint: disable=too-many-public-methods
-
-	@staticmethod
-	def create_embedded(factory):
-		return factory.create_embedded
-
-	@staticmethod
-	def create(factory):
-		return factory.create
-
-	# region create
+class SymbolTransactionFactoryTest(BasicTransactionFactoryTest):
+	# pylint: disable=no-member
+	# pylint: disable=abstract-method
```

Comment:
why these

Reply:
cause this isn't actual impl (they are at the bottom and they do inherit TestCase), but pylint can't figure that out.
supposedly known issue:
https://stackoverflow.com/questions/39256350/pylint-cannot-handle-abstract-subclasses-of-abstract-base-classes

Reply:
no-member makes sense but why abstract-method ?

Reply:
abstract-method is for methods that are not overridden, like so:
`test_TransactionFactory.py:17:0: W0223: Method '_assert_signature' is abstract in class 'BasicTransactionFactoryTest' but is not overridden (abstract-method)`

(no-member is for "missing" TestCase things like self.assertEqual)